docs/labwc-config.5.scd

```diff
@@ -844,6 +844,9 @@ defined as shown below.
 
 	*title* is the title of the window.
 
+	*type* relates to NET_WM_WINDOW_TYPE for XWayland cleints and internal
+	heuristics for native Wayland windows.
```

Comment:
`s/cleints/clients/`

Reply:
That's a long list.

Reply:
I'd just add the whole list for now. We can always update it later.
Without that list the feature is basically useless without reading the code.